lib/views/cgu/cgu_page.dart

```diff
@@ -92,7 +92,7 @@ class _CguPageState extends State<CguPage> {
                           type: LenraComponentType.secondary,
                           onPressed: () {
                             authModel.logout().then((value) {
-                              CommonNavigator.go(context, CommonNavigator.login);
+                              CommonNavigator.go(context, CommonNavigator.sign, extra: {"register": false});
```

Comment:
It might be nice to do not set the extra and define a default value when it's not given.